specification/apimanagement/control-plane/Microsoft.ApiManagement/preview/2017-03-01/apimemailtemplate.json

```diff
@@ -0,0 +1,407 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ApiManagementClient",
+    "description": "Use these REST APIs for performing operations on Email Templates associated with your Azure API Management deployment.",
+    "version": "2017-03-01"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{apimBaseUrl}",
+    "useSchemePrefix": false,
+    "positionInOperation": "first",
+    "parameters": [
+      {
+        "name": "apimBaseUrl",
+        "description": "The management endpoint of the API Management service, for example https://myapimservice.management.azure-api.net.",
+        "required": true,
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "apim_key": {
+      "type": "apiKey",
+      "name": "Authorization",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apim_key": []
+    }
+  ],
+  "paths": {
+    "/templates": {
+      "get": {
+        "tags": [
+          "EmailTemplate"
+        ],
+        "operationId": "EmailTemplate_List",
```

Comment:
>EmailTemplate_List [](start = 24, length = 18)

No odata extension?

Reply:
let me check

---
In reply to: [163354744](https://github.com/Azure/azure-rest-api-specs/pull/2269#discussion_r163354744) [](ancestors = 163354744)

Reply:
fixed